resources/wren/shaders/overlay.frag

```diff
@@ -29,7 +29,7 @@ layout(std140) uniform Overlay {
   vec4 activeFlags;   // x: bg texture, y: main texture, z: fg texture, w: border
   vec4 textureFlags;  // x: flip vertically, y: additional texture count, z: maxRange (depth textures only),
                       // w: overlay transparency
-  vec4 sizeInPixels;  // x,y: size in screen pixels, z: screen ratio, w: unused (std140 padding)
+  vec2 sizeInPixels;  // x,y: size in screen pixels
   vec2 borderSize;    // x: vertical size, y: horizontal size
```

Comment:
I am wondering if it is not worth to combine these two vec2 for performance.

Reply:
Yes, thank you.

Reply:
No it's ok as-is. This respects std140 layouts (a multiple of 4 floats).

Reply:
Yes, exactly.